process/testing/winfv-felix/calico-node-windows.yaml

```diff
@@ -7,21 +7,21 @@ spec:
           env:
             - name: FELIX_HEALTHENABLED
               value: "true"
-          image: docker.io/library/node-windows:latest
+          image: quay.io/library/node-windows:latest
```

Comment:
this file actually points to images that are loaded locally (`docker.io/library`). I'm not sure if it's work with `quay.io`, but if the Win CAPZ CI fails, we should keep `docker.io` in this file...
cc @MichalFupso

Reply:
@coutinhop  `this file actually points to images that are loaded locally` What it means?

Reply:
we build those images locally, export them as tarballs, `scp` into the node and import them there. here's the import part:
https://github.com/projectcalico/calico/blob/master/process/testing/winfv-felix/setup-fv-capz.sh#L110-L117